src/clientsettingsdlg.cpp

```diff
@@ -862,6 +861,9 @@ void CClientSettingsDlg::UpdateSoundDeviceChannelSelectionFrame()
         // input
         cbxLInChan->clear();
         cbxRInChan->clear();
+
+        int iSndChanIdx;
+
         for ( iSndChanIdx = 0; iSndChanIdx < pClient->GetSndCrdNumInputChannels(); iSndChanIdx++ )
```

Comment:
As this variable is only used within the loops, it doesn't need to be declared outside the `for()` at all:
```suggestion

        for ( int iSndChanIdx = 0; iSndChanIdx < pClient->GetSndCrdNumInputChannels(); iSndChanIdx++ )
```
And also on line 881 below (Github wouldn't let me attach a suggestion to that line)

Reply:
Probably yes? I still don’t understand why it was that high out

Reply:
Neither do I. Maybe it was previously used outside of the loops, although it isn't now.